addons/discovery-app/README.md

```diff
@@ -0,0 +1,10 @@
+# discovery-app addon
+
+This template is tailored on Discivery application deployment and it's meant to me applied after the (simple-app)[../../bases/simple-app] template
```

Comment:
```suggestion
This template is tailored on Discovery application deployment and it's meant to me applied after the (simple-app)[../../bases/simple-app] template
```

Reply:
Not sure it's meant to be run after the simple-app because of the github workflow overwrittin, and the same for the deploy/app terraform files and environments/* files... 
Or it should be in a deploy/app2 folder

In fact, this addon is more like a bases than an addon

Reply:
Not really:
- The addons has just some code, that will be applied on top of the repo (where we have already deployed the simple app)
- Avoiding to have a whole new base means we can avoid maintaing duplicated code, for ex the infra sub-stack
 - The addon would create a branch (not a PR) proposing changes to specific files. We can make adjustments before opening the PR. This would be the case even with a second app folder (for ex. app2)